src/azure-cli/azure/cli/command_modules/acs/_resourcegroup.py

```diff
@@ -0,0 +1,17 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
This file is basically the same as the [_resourcegroup.py](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/_resourcegroup.py) file in aks-preview.

The differences are as follows:
- sort imports
- change the error type

For reference, [this](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/custom.py#L4154) is the original implementation in `custom.py`.